pyproject.toml

```diff
@@ -6,27 +6,26 @@ authors = ["Kevin Howbrook <kevin.howbrook@torchbox.com>"]
 
 [tool.poetry.dependencies]
 python = "~3.8"
-django = "~3.2"
-wagtail = "~4.1"
+django = "~3.2.20"
+wagtail = "~5.0"
 psycopg2 = "~2.8"
 gunicorn = {version = "~20.0", optional = true}
-wagtail-django-recaptcha = "1.0"
+wagtail-django-recaptcha = {git = "https://github.com/torchbox-forks/wagtail-django-recaptcha", branch = "support/wagtail-50"}
```

Comment:
**suggestion**: It'd be better to use a stable target rather than a branch reference. Could the changes for this app (and _wagtail-storages_ below) be given a release/tag to use instead?

Reply:
Created tags and replaced usage of branch.